app/Resources/static/themes/_global/index.js

```diff
@@ -107,4 +109,12 @@ $(document).ready(() => {
   $('#user_emailTwoFactor').on('change', () => {
     $('#user_googleTwoFactor').prop('checked', false);
   });
+
+  // handle copy to clipboard for developer stuff
+  const clipboard = new ClipboardJS('.btn');
+  clipboard.on('success', (e) => {
+    console.info(e.text);
```

Comment:
can be remove for prod?

Reply:
I put that because I can't only use:

```js
new ClipboardJS('.btn');
```

Because ESLint don't allow it, see https://eslint.org/docs/rules/no-new
So I've to store it in a variable, but then I need to use that variable (otherwise ESLint isn't happy again). So I decided to use the `.on('success'` to use the variable

:man_shrugging:

Reply:
console.info is required ?